PredixMobileUIKit/PredixBarChartView.swift

```diff
@@ -0,0 +1,330 @@
+//
+//  PredixBarChartView.swift
+//  PredixMobileUIKit
+//
+//  Created by Fouche, George (GE Healthcare) on 11/15/17.
+//  Copyright © 2017 GE. All rights reserved.
+//
+
+import Charts
+import Foundation
+import PredixSDK
+
+// MARK: - The Option Enum
+
+/// List of options that can be used on the Bar Chart.
+public enum Option {
+    /// toggle on or off, Bars value on the chart.
```

Comment:
Capitalize the first word of code documentation lines. 
Also, don't use a period unless the code comment is a full sentence with multiple sentences in the comment.
Consider this for all code comments, as it's common throughout.